bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -7,4 +7,18 @@
 	<name>@text/bindingName</name>
 	<description>@text/bindingDescription</description>
 	<connection>local</connection>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_http._tcp.local.</mdns-service-type>
+			<match-properties>
+				<match-property>
+					<name>hostname</name>
```

Comment:
Testing again with TRACE:
```
09:28:27.620 [TRACE] [on.finders.MDNSAddonSuggestionFinder] - mDNS service name=FP00112A22751D._http._tcp.local., properties=[]
```

So it looks like the property should be `name` after all.

Reply:
If you are using my build from yesterday, I think there was a bug whereby the trace log was not showing the properties. I made a fix of the logging last night, but I need to test again.

Reply:
I am running in Eclipse, not your build from yesterday. I pulled updates of all PR's this morning.

Reply:
So the logging is apparently still not working. Or did you see other log entries where the properties list IS populated?

Reply:
I also so this:
```
09:28:28.160 [TRACE] [on.finders.MDNSAddonSuggestionFinder] - mDNS service name=SCC-00000874._http._tcp.local., properties=[SCC-00000874._http._tcp.local.=true]
09:28:28.161 [TRACE] [on.finders.MDNSAddonSuggestionFinder] - mDNS service name=HP Color LaserJet MFP M281fdw (0CACA4)._http._tcp.local., properties=[HP Color LaserJet MFP M281fdw (0CACA4)._http._tcp.local.=true]
```

Reply:
Yes the properties log listing is still not showing the real actual properties. I am working on it..

Reply:
There seems to be a thread synchronization issue with the JMDNS getPropertyNames Enumeration (and/or the parallelism of stream operations) which makes the logging difficult. I am really struggling with it. So I may just remove some of the logging again.